packages/cli/doc-core/src/node/runtimeModule/globalStyles.ts

```diff
@@ -11,7 +11,7 @@ export async function globalStylesVMPlugin(context: FactoryContext) {
     ...globalStylesByPlugins,
   ]
     .filter(source => source.length > 0)
-    .map(source => `import '${source}';`)
+    .map(source => `import ${JSON.stringify(source)};`)
```

Comment:
So what is the difference between `'${source}'` and `JSON.stringify(source)`, can you provide some demo code?

Reply:
I'll add a demo in  the details.

Reply:
okk